src/main/scala/beam/agentsim/agents/PersonAgent.scala

```diff
@@ -749,6 +749,26 @@ class PersonAgent(
         )
       )
       goto(WaitingForReservationConfirmation)
+    // CAV but too late
+    // TODO: Refactor so it uses literally the same code block as transit
+    case Event(StateTimeout, data @ BasePersonData(_, _, nextLeg :: _, _, _, _, _, _, _, _, _))
+        if nextLeg.beamLeg.startTime < _currentTick.get =>
+      // We've missed the CAV. This occurs when something takes longer than planned (based on the
+      // initial inquiry). So we replan but change tour mode to WALK_TRANSIT since we've already done our non-transit
+      // portion.
+      log.error("Missed CAV pickup, late by {} sec", _currentTick.get - nextLeg.beamLeg.startTime)
```

Comment:
Curious, should this be a warning, or even less? Does it really warrant error? Error to me means there is a bug in something (could be config vs code) and this is here to notify us - my two cents at least

Reply:
I agree @JustinPihony, can you make this a WARN on the RH depot charging branch?

Reply:
Yes, agree, too. I _think_ I made this an error because before this fix it used to be, .. well, an actual exception, and I thought maybe we still want to know how often this happens for a short while.